qa/workunits/fs/misc/osd_full_mgr_hang.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh -x
+
+set -e
+
+ceph fs subvolume create cephfs sub_0
+subvol_path=$(ceph fs subvolume getpath cephfs sub_0 2>/dev/null)
+
+#For debugging
+echo "Before write"
+df -h
+ceph osd df
+
+sudo dd if=/dev/urandom of=$CEPH_MNT$subvol_path/500MB_file-1 status=progress bs=1M count=500
+
+#To reflect in osd df output
+sleep 5
+
+#For debugging
+echo "After write"
+df -h
+ceph osd df
+
+ceph osd set-full-ratio 0.2
+ceph osd set-nearfull-ratio 0.16
+ceph osd set-backfillfull-ratio 0.18
+
+#For debugging
+echo "After ratio set"
+df -h
+ceph osd df
+
+#Sleep 5 seconds to reflect osd config settings
+sleep 5
+
+ceph fs subvolume rm cephfs sub_0
+
+#For subvolume to actually remove subvolume data
+sleep 10
+sudo rm -rf $CEPH_MNT/volumes
```

Comment:
This is weird: shouldn't you just be checking that the subvolume trash directory is empty and that the subvolume does not exist?

Reply:
Done